cmd/protoc-gen-gogo/protoc-gen-gogo.go

```diff
@@ -0,0 +1,42 @@
+/*
+Copyright 2015 The Kubernetes Authors All rights reserved.
```

Comment:
This is still wrong, right one: https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/kubelet.go#L1-L15